services/frontend-v3/src/components/basicInfo/BasicInfoView.jsx

```diff
@@ -10,20 +10,20 @@ function BasicInfoView(props) {
   /* Component Styles */
   const styles = {
     profileHeaderRow: {
-      margin: "25px 0",
```

Comment:
I think we decided to leave these commas in. As trailing commas are recommended by some people to make diffs easier. We should discuss this with Risvi further.

Reply:
I agree. Maybe your version of Prettier or VS Code is not up to date, Kat.

Reply:
maybe we should add a prettier config file to resolve this issue in the future?

Reply:
I fixed this- my prettier was out of date sorry guys!